packages/core/types/index.js

```diff
@@ -133,6 +133,7 @@ export type PackageTargetDescriptor = {|
   +isLibrary?: boolean,
   +optimize?: boolean,
   +scopeHoist?: boolean,
+  +source?: string,
```

Comment:
Should be `string | Array<string>`?

Reply:
Seems it should be consistent with this:
https://github.com/parcel-bundler/parcel/blob/3b709981c5a15aa11a5c9998ad719b4d36d42636/packages/core/types/index.js#L237

Isn't that one wrong though. I'm not super familiar with flow types, but shouldn't that be `FilePath | Array<FilePath>`

Reply:
Yeah looks like they should both be updated. 😬

Reply:
I updated both. This caused a flow error in NodeResolver.js which I fixed but I want some more experienced eyes on.